src/strings/en-us.json

```diff
@@ -1625,5 +1625,38 @@
+    "LabelTrickplayAccel": "Enable hardware acceleration",
+    "LabelTrickplayAccelHelp": "Make sure to enable 'Allow MJPEG Encoding' in Transcoding if your hardware supports it.",
+    "NonBlockingScan": "Non Blocking - queues generation, then returns",
+    "BlockingScan": "Blocking - queues generation, blocks scan until complete",
+    "LabelScanBehavior": "Scan Behavior",
+    "LabelScanBehaviorHelp": "The default behavior is non blocking, which will add media to the library before trickplay generation is done. Blocking will ensure trickplay files are generated before media is added to the library, but will make scans significantly longer.",
+    "PriorityHigh": "High",
+    "PriorityAboveNormal": "Above Normal",
+    "PriorityNormal": "Normal",
+    "PriorityBelowNormal": "Below Normal",
+    "PriorityIdle": "Idle",
+    "LabelProcessPriority": "Process Priority",
+    "LabelProcessPriorityHelp": "Setting this lower or higher will determine how the CPU prioritizes the ffmpeg trickplay generation process in relation to other processes. If you notice slowdown while generating trickplay images but don't want to fully stop their generation, try lowering this as well as the thread count.",
+    "LabelImageInterval": "Image Interval",
+    "LabelImageIntervalHelp": "Interval of time (ms) between each new trickplay image.",
+    "LabelWidthResolutions": "Width Resolutions",
+    "LabelWidthResolutionsHelp": "Comma separated list of the widths (px) that trickplay images will be generated at. All images should generate proportionally to the source, so a width of 320 on a 16:9 video ends up around 320x180.",
+    "LabelTileWidth": "Tile Width",
+    "LabelTileWidthHelp": "Maximum number of images per tile in the X direction.",
+    "LabelTileHeight": "Tile Height",
+    "LabelTileHeightHelp": "Maximum number of images per tile in the Y direction.",
+    "LabelJpegQuality": "JPEG Quality",
+    "LabelJpegQualityHelp": "The JPEG compression quality for trickplay images.",
+    "LabelQscale": "Qscale",
+    "LabelQscaleHelp": "The quality scale of images output by ffmpeg, with 2 being the highest quality and 31 being the lowest.",
+    "LabelTrickplayThreads": "FFmpeg Threads",
+    "LabelTrickplayThreadsHelp": "The number of threads to pass to the '-threads' argument of ffmpeg.",
+    "OptionExtractTrickplayImage": "Enable trickplay image extraction",
+    "ExtractTrickplayImagesHelp": "Trickplay images are similar to chapter images, except they span the entire length of the content and are used to show a preview when scrubbing through videos.",
+    "LabelExtractTrickplayDuringLibraryScan": "Extract trickplay images during the library scan",
+    "LabelExtractTrickplayDuringLibraryScanHelp": "Generate trickplay images when videos are imported during the library scan. Otherwise, they will be extracted during the chapter images scheduled task. If generation is set to non-blocking this will not affect the time a library scan takes to complete."
```

Comment:
It seems there is a dedicated scheduled task for extracting trickplay images. Should this be referencing the chapter image task?

Reply:
I'm not the pull request author (apologies for the drive-by comment; I'm just excited for this 10.9 feature), but I believe this and the corresponding `jellyfin/jellyfin` PR that was previously merged are based on the [Jellyscrub](https://github.com/nicknsy/jellyscrub) plugin. This plug-in features a background task for generating BIF previews, which is separate from the existing chapter image preview task. I'm not sure if this is what is being referenced by this particular string?

Reply:
> It seems there is a dedicated scheduled task for extracting trickplay images. Should this be referencing the chapter image task?

These string specifically are for the library settings that allow you to enable/disable trickplay generation and generation on scan per library. This is very similar to chapter images however trickplay images take much longer to generate and are functionally a bit different since they have no concept of segments and always span the length of the media (whereas chapters have a start and stop time).

On the server, the [trickplay task](https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Providers/Trickplay/TrickplayImagesTask.cs) is also separate from chapter images so users can choose to enable/disable whichever they prefer.